### DISCUSSION TITLE
about LEO orbit determination

### DISCUSSION THREAD

**Discussion by chenxi202** [groops-devs/groops#230] [General]
When I use reduced dynamical to determine LEO satellite orbit (scenarioLeoReducedDynamicOrbit), of course not the same data as the example。 There is a mistake，as shown in the figure below ，I don't know why. But when I use BERNESE to deal with the same data, it can work well. So, maybe somewhere I'm configuring incorrectly. If anyone encounters this problem, please don't hesitate to advise！
![7](https://github.com/groops-devs/groops/assets/152595071/d1d8a564-38ee-47d6-af92-36ee2aea9eee)

**Comment by Waley04** [comment]
Hey @chenxi202,

Could you please share the sample data so that I can try to replicate the issue?

Thanks,

Waley.

**Reply by chenxi202** [reply]
Hey, @Waley04. The attachment contains my entire task package, including data（I specifically removed one hour from the observation file to create a gap）  and configuration files (xml file). 

When I set minGap is 3600, there is a result like the figure below 
<img width="505" alt="78628032f76e89b1c77c82eb6d9aade" src="https://github.com/groops-devs/groops/assets/152595071/975a68a3-39d5-472b-943b-3c4c012a0920">
<img width="735" alt="f0768226b9faf2b400974b79a9271f3" src="https://github.com/groops-devs/groops/assets/152595071/5cdfca5e-4b78-4d40-a9ef-d98722120771">

so, 04groopsGnssProcessing-float.xml must be configure wrong.

Besides, if I set minGap is 0(what I think is wrong), there is a result like the figure below.
<img width="724" alt="28ed42831c96e85bd05d99b3e9c5fe7" src="https://github.com/groops-devs/groops/assets/152595071/e0340c7c-4787-4c8f-abf9-daeac87bd9ef">

So, can you Can you point out my mistake？
Thank you very  much ~~~

**Reply by chenxi202** [reply]
The attachment file is so large that the upload fails，so can I have you email address so that I can send it to you?

**Comment by tmayerguerr** [comment]
Hi,

As stated on the screen: `SENTINEL6a disabled: not enough estimable epochs`.
Maybe it helps to adjust `minEstimableEpochsRatio` in [Parametrization:html#leoDynamicOrbits](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#leoDynamicOrbits).

regards
Torsten

**Comment by chenxi202** [comment]
Thank you very much, I 'll try it now

2023-12-02 20:21:48"Torsten Mayer-Guerr" ***@***.***>写道：

Hi,

As stated on the screen: SENTINEL6a disabled: not enough estimable epochs.
Maybe it helps to adjust minEstimableEpochsRatio in Parametrization:html#leoDynamicOrbits.

regards
Torsten

**Reply by chenxi202** [reply]
Hi,
According to your advice， I have set the "minEstimableEpochsRatio in Parametrization:html#leoDynamicOrbits" as 0.1, but it dosen't work either. So, I still need to modify where?
Thank you again for you suggestioin.

regards
Chenxi
